src/components/crafts-table/index.js

```diff
@@ -387,15 +425,51 @@ function CraftTable(props) {
             {
                 Header: t('Cost ₽'),
                 accessor: (d) => Number(d.cost),
-                Cell: ValueCell,
+                Cell: (props) => {
+                    if (props.row.original.cached) {
+                        return (
+                            <div className="center-content">
+                                <Tippy
+                                    placement="bottom"
+                                    content={t('Flea market prices loading')}
+                                >
+                                    <Icon
+                                        path={mdiTimerSand}
+                                        size={1}
+                                        className="icon-with-text"
+                                    />
+                                </Tippy>
+                            </div>
+                        );
+                    }
+                    return <ValueCell value={props.value} valueCount={props.row.original.reward.count}/>;
+                },
                 id: 'cost',
             },
             ...(includeFlea
                 ? [
                       {
                           Header: t('Flea throughput/h'),
                           accessor: 'fleaThroughput',
-                          Cell: ValueCell,
+                          Cell: (props) => {
+                            if (props.row.original.cached) {
+                                return (
+                                    <div className="center-content">
+                                        <Tippy
+                                            placement="bottom"
+                                            content={t('Flea market prices loading')}
+                                        >
+                                            <Icon
+                                                path={mdiTimerSand}
+                                                size={1}
+                                                className="icon-with-text"
+                                            />
+                                        </Tippy>
+                                    </div>
```

Comment:
As this is repeated so many times it would be nice to refactor it in a separate function

Reply:
Good idea, I've done that.